frontend/apps/app/features/migrations/actions/resolveReviewFeedback.ts

```diff
@@ -37,7 +69,54 @@ export const resolveReviewFeedback = async (data: {
       throw new Error(`Failed to resolve feedback: ${error.message}`)
     }
 
-    return { success: true, data: updatedFeedback }
+    // Create a knowledge suggestion based on the feedback
+    const overallReviewId = feedbackData.overallReviewId
+    if (!overallReviewId) {
+      throw new Error('Overall review ID not found for the feedback')
+    }
```

Comment:
This check seems unnecessary because `overallReviewId` is not-null:

https://liam-erd-web.vercel.app/app/projects/6/ref/main/schema/frontend/packages/db/schema/schema.sql?active=ReviewFeedback&showMode=ALL_FIELDS

Reply:
Thank you! I have correct it.
https://github.com/liam-hq/liam/pull/1371/commits/4e58400dfdb092bdd06c23336057fd275bf7bd56